src/main/target/AIKONF4/target.h

```diff
@@ -0,0 +1,143 @@
+/*
+ * This is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define TARGET_BOARD_IDENTIFIER "AIK4"
+#define USBD_PRODUCT_STRING "AIKONF4"
+
+#ifdef OPBL
+#define USBD_SERIALNUMBER_STRING "0x8020000" // Remove this at the next major release (?)
+#endif
+
+#define LED0_PIN                  PB4
+#define BEEPER                    PB5
+#define BEEPER_INVERTED
+
+#define USE_DSHOT_DMAR
```

Comment:
Not needed.
I think you want #define ENABLE_DSHOT_DMAR true instead.

Reply:
Thanks
Indeed, it was a bit outdated, from the days before `dshot_burst` CLI setting